articles/node-js-vs-django/index.md

```diff
@@ -0,0 +1,83 @@
+3. Object-oriented – It can model real-world data rather than functions and codes.
+4. It is swift to execute codes.
+5. A lot of community tutorials, resources, and shared codes are available.
+
+### Django
+
+It is an open-source Python web framework primarily designed to ensure high-speed web development. It is easy to learn for those knowledgeable in Python. Django has strict compliance with the rules and formalized scenarios for project implementation. Django provides accuracy in coding, a straight-forward app architecture, fewer errors, and bugs in the output.
+
+#### Basic features of Django
+
+1. Simplicity – Especially with the Python language, Django is easy to learn, use, and implement.
+2. It is an open-source language – No fees for installation and usage. It is available and freely accessible to the public.
+3. High-level language – It simplifies computer syntax into a language the user can easily understand.
+4. It is both object-oriented and functional – meaning it can model real-world data and focus on functions. Node.js allows code re-useability.
+5. Portable – There is no need to write different codes for different machines as it is highly portable and flexible.
+6. It is available with a vast collection of libraries – Libraries are inbuilt, and therefore, there is no need to write unique code for everything.
+
+### Comparison between Django and Node.js
+
+| Distinctive point | Django | Node.js |
+| --- | --- | --- |
+| Programming language and primary use | It is a [Python based framework](https://www.upgrad.com/blog/python-frameworks/) used by developers to build computer apps within a stipulated time frame.| A framework that runs on JavaScript. Developers use it to build robust application programming interfaces (client-sided). |
+| Architecture | Follows the [MTV model template](https://www.javatpoint.com/django-mvt) for purposes of handling data, validating, and interaction. | A runtime environment, therefore, works on an [event-driven model](https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture). Runs on any operating system, maintaining a small number of requests. |
+| Security | It is more secure, comes with a built-in system, preventing a security breach. | It requires manual operations systems to manage security risks. |
+| Performance | It has a built-in house template system; hence the execution of tasks is quicker, thereby offering better performance. | It allows web professionals freedom in implementations. Though, it has a limitation of increasing the time required to build an application. |
+| Complexity | It is more complicated as a developer has to follow a particular specified path for solving problems. | It is less complicated as a developer has a free hand to solve problems. |
+| Efficiency | More efficient and faster, hence cost-effective. | It consumes more operation time; hence it is a less cost-effective option. |
```

Comment:
This is debatable. For example, Customizability with Django is limited when compared to JS-based frameworks. Therefore, check this fact once.

Reply:
> This is debatable. For example, Customizability with Django is limited when compared to JS-based frameworks. Therefore, check this fact once.

@lalith1403 please be specific on the issue, as I have not touched on customization with Django in the article.

Reply:
I was referring to cost effectiveness. Cost effectiveness versus quality is my point.

Reply:
@lalith1403 Thank you for your guidance, my argument is based on the fact that Django has in-built libraries and processes, a developer would not start from square one as it is for Node.js; hence less time, and therefore cost effective.

I understand your argument and I can see risk of misleading to the reader, let me just do away with that point.